libr/arch/p/pyc/opcode_arg_fmt.c

```diff
@@ -22,20 +22,20 @@ char *format_CALL_FUNCTION_KW_36(ut32 oparg) {
 }
 
 char *format_CALL_FUNCTION_EX_36(ut32 oparg) {
-	return r_str_new ((oparg & 0x01)? "keyword args": "");
+	return R_STR_DUP ((oparg & 0x01)? "keyword args": "");
```

Comment:
```suggestion
	return strdup ((oparg & 0x01)? "keyword args": "");
```

dont use R_STR_DUP when the argument cant be null